src/RA_Json.h

```diff
@@ -11,6 +14,9 @@
 #include <rapidjson\ostreamwrapper.h>
 #include <rapidjson\error\en.h>
 
-extern void _WriteBufferToFile(const std::wstring& sFileName, const rapidjson::Document& doc);
+void _WriteBufferToFile(const std::wstring& sFileName, const rapidjson::Document& doc);
+
+bool LoadDocument(_Out_ rapidjson::Document& doc, ra::services::TextReader& reader);
+bool SaveDocument(_In_ rapidjson::Document& doc, ra::services::TextWriter& writer);
```

Comment:
If doc is an "in" parameter it should be marked with `const `since it's a reference.

Reply:
will do